<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="0" failed="5" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-07-12T23:59:41 IST" name="Default suite" finished-at="2022-07-12T23:59:42 IST" duration-ms="971">
    <groups>
    </groups>
    <test started-at="2022-07-12T23:59:41 IST" name="Default test" finished-at="2022-07-12T23:59:42 IST" duration-ms="971">
      <class name="tests.ProductTest">
        <test-method is-config="true" signature="preSetup()[pri:0, instance:tests.ProductTest@95bb2a2]" started-at="2022-07-12T23:59:41 IST" name="preSetup" finished-at="2022-07-12T23:59:41 IST" duration-ms="9" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preSetup -->
        <test-method is-config="true" signature="setup()[pri:0, instance:tests.ProductTest@95bb2a2]" started-at="2022-07-12T23:59:41 IST" name="setup" finished-at="2022-07-12T23:59:41 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateRequest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:tests.ProductTest@95bb2a2]" started-at="2022-07-12T23:59:41 IST" name="validateRequest" data-provider="DataFromExcel" finished-at="2022-07-12T23:59:42 IST" duration-ms="805" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[GET]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[/products]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA["content-type": "application/json; charset=utf-8",
"Content-Encoding": "gzip"]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[NA]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[$limit,3]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[200.0]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[data[0].name="Duracell - AAA Batteries (4-Pack)",
data[1].name="Duracell - AA 1.5V CopperTop Batteries (4-Pack)",
data[2].name="Duracell - AA Batteries (8-Pack)"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [null] but found [ "application/json; charset=utf-8"]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [null] but found [ "application/json; charset=utf-8"]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at request.RequestMethods.validateHeaders(RequestMethods.java:29)
at request.RequestMethods.getProduct(RequestMethods.java:51)
at tests.ProductTest.validateRequest(ProductTest.java:25)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateRequest -->
        <test-method signature="validateRequest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:tests.ProductTest@95bb2a2]" started-at="2022-07-12T23:59:42 IST" name="validateRequest" data-provider="DataFromExcel" finished-at="2022-07-12T23:59:42 IST" duration-ms="49" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[POST]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[/products]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA["content-type": "application/json; charset=utf-8",
"Content-Encoding": "gzip"]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[{
"name": "iphone",
"type": "Smart Phone",
"price": 1000,
"shipping": 10,
"upc": "string",
"description": "color white, ram 4 gb, processor A12 bionic",
"manufacturer": "Apple California",
"model": "iphone 11",
"url": "string",
"image": "string"
}]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[NA]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[201.0]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[NA]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [null] but found [ "application/json; charset=utf-8"]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [null] but found [ "application/json; charset=utf-8"]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at request.RequestMethods.validateHeaders(RequestMethods.java:29)
at request.RequestMethods.postProduct(RequestMethods.java:68)
at tests.ProductTest.validateRequest(ProductTest.java:29)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateRequest -->
        <test-method signature="validateRequest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:tests.ProductTest@95bb2a2]" started-at="2022-07-12T23:59:42 IST" name="validateRequest" data-provider="DataFromExcel" finished-at="2022-07-12T23:59:42 IST" duration-ms="44" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[PUT]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[/products]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA["content-type": "application/json; charset=utf-8",
"Content-Encoding": "gzip"]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[{
"name": "iphone",
"type": "Smart Phone",
"price": 1000,
"shipping": 10,
"upc": "string",
"description": "color white, ram 4 gb, processor A12 bionic",
"manufacturer": "Apple California",
"model": "iphone 11",
"url": "string",
"image": "string"
}]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[id]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[200.0]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[NA]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.ClassCastException">
            <message>
              <![CDATA[class io.restassured.path.xml.XmlPath cannot be cast to class java.lang.Integer (io.restassured.path.xml.XmlPath is in unnamed module of loader 'app'; java.lang.Integer is in module java.base of loader 'bootstrap')]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ClassCastException: class io.restassured.path.xml.XmlPath cannot be cast to class java.lang.Integer (io.restassured.path.xml.XmlPath is in unnamed module of loader 'app'; java.lang.Integer is in module java.base of loader 'bootstrap')
at request.RequestMethods.putProduct(RequestMethods.java:82)
at tests.ProductTest.validateRequest(ProductTest.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ClassCastException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateRequest -->
        <test-method signature="validateRequest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:tests.ProductTest@95bb2a2]" started-at="2022-07-12T23:59:42 IST" name="validateRequest" data-provider="DataFromExcel" finished-at="2022-07-12T23:59:42 IST" duration-ms="17" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[PATCH]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[/products]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA["content-type": "application/json; charset=utf-8",
"Content-Encoding": "gzip"]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[{ "name": "iphone",
"type": "Smart Phone", 
"price": 1500,
"shipping": 15, 
"upc": "string", 
"description": "color green, ram 8 gb, processor A15 bionic", 
"manufacturer": "Apple California", 
"model": "iphone 13", 
"url": "string", 
"image": "string" 
}]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[id]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[201.0]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[NA]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.ClassCastException">
            <message>
              <![CDATA[class io.restassured.path.xml.XmlPath cannot be cast to class java.lang.Integer (io.restassured.path.xml.XmlPath is in unnamed module of loader 'app'; java.lang.Integer is in module java.base of loader 'bootstrap')]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ClassCastException: class io.restassured.path.xml.XmlPath cannot be cast to class java.lang.Integer (io.restassured.path.xml.XmlPath is in unnamed module of loader 'app'; java.lang.Integer is in module java.base of loader 'bootstrap')
at request.RequestMethods.patchProduct(RequestMethods.java:104)
at tests.ProductTest.validateRequest(ProductTest.java:37)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ClassCastException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateRequest -->
        <test-method signature="validateRequest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:tests.ProductTest@95bb2a2]" started-at="2022-07-12T23:59:42 IST" name="validateRequest" data-provider="DataFromExcel" finished-at="2022-07-12T23:59:42 IST" duration-ms="13" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[DELETE]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[/products]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA["content-type": "application/json; charset=utf-8",
"Content-Encoding": "gzip"]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[{
"name": "iphone",
"type": "Start Phone",
"price": 1000,
"shipping": 10,
"upc": "string",
"description": "color white, ram 4 gb, processor A12 bionic",
"manufacturer": "Apple California",
"model": "iphone 11",
"url": "string",
"image": "string"
}]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[id]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[200400.0]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[NA]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.ClassCastException">
            <message>
              <![CDATA[class io.restassured.path.xml.XmlPath cannot be cast to class java.lang.Integer (io.restassured.path.xml.XmlPath is in unnamed module of loader 'app'; java.lang.Integer is in module java.base of loader 'bootstrap')]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ClassCastException: class io.restassured.path.xml.XmlPath cannot be cast to class java.lang.Integer (io.restassured.path.xml.XmlPath is in unnamed module of loader 'app'; java.lang.Integer is in module java.base of loader 'bootstrap')
at request.RequestMethods.deleteProduct(RequestMethods.java:122)
at tests.ProductTest.validateRequest(ProductTest.java:40)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ClassCastException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateRequest -->
      </class> <!-- tests.ProductTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
